src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -199,6 +199,7 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
             }
         }
 
+        [PlatformSpecific(PlatformID.Windows)] // CopyToAsync(Stream, TransportContext) isn't used on unix
```

Comment:
"isn't used".  We call ReadAsStreamAsync() (like the WinRT implementation) rather than calling CopyToAsync(Stream, TransportContext) (like the WinHttp implementation)